LoopBackTests/server/index.js

```diff
@@ -132,10 +129,10 @@ Widget.destroyAll(function () {
   });
 });
 
-app.model(loopback.AccessToken);
+app.model(loopback.AccessToken, {dataSource: 'Memory'});
```

Comment:
This should be automatically handled by `app.enableAuth({ dataSource: 'Memory' });`.